backstopper-jaxrs/src/main/java/com/nike/backstopper/handler/jaxrs/JaxRsApiExceptionHandler.java

```diff
@@ -0,0 +1,116 @@
+package com.nike.backstopper.handler.jaxrs;
+
+import com.nike.backstopper.apierror.ApiError;
+import com.nike.backstopper.apierror.projectspecificinfo.ProjectApiErrors;
+import com.nike.backstopper.handler.*;
+import com.nike.backstopper.handler.jaxrs.config.JaxRsDefaultExceptionHandlers;
+import com.nike.backstopper.handler.listener.ApiExceptionHandlerListener;
+import com.nike.backstopper.model.DefaultErrorContractDTO;
+import com.nike.backstopper.model.util.JsonUtilWithDefaultErrorContractDTOSupport;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * An {@link ApiExceptionHandlerServletApiBase} extension that hooks into JAX-RS via its
+ * {@link ExceptionMapper} implementation, specifically {@link ExceptionMapper#toResponse(Throwable)}.
+ *
+ * <p>Any errors not handled here are things we don't know how to deal with and will fall through to {@link
+ * JaxRsUnhandledExceptionHandler}.
+ *
+ * @author Michael Irwin
+ */
+@Provider
+@Singleton
+public class JaxRsApiExceptionHandler extends ApiExceptionHandlerServletApiBase<Response.ResponseBuilder>
+    implements ExceptionMapper<Throwable> {
+
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    /**
+     * The {@link JaxRsUnhandledExceptionHandler} that handles any
+     * exceptions we weren't anticipating
+     */
+    @SuppressWarnings("WeakerAccess")
+    protected final JaxRsUnhandledExceptionHandler jerseyUnhandledExceptionHandler;
+
+    @Context
+    protected HttpServletRequest request;
+
+    @Context
+    protected HttpServletResponse response;
+
+    @Inject
+    public JaxRsApiExceptionHandler(ProjectApiErrors projectApiErrors,
+                                    JaxRsDefaultExceptionHandlers apiExceptionHandlerListeners,
```

Comment:
The exception handler list injected should be conceptually whatever the app wants to use - it's not just the defaults. I would recommend renaming `JaxRsDefaultExceptionHandlers` to `JaxRsApiExceptionHandlerListenerList` and renaming the `Jersey2BackstopperConfigHelper.ApiExceptionHandlerListenerList` inner class to `Jersey2ApiExceptionHandlerListenerList`.